packages/rocketchat-cors/cors.js

```diff
@@ -3,11 +3,18 @@ import _ from 'underscore';
 
 import url from 'url';
 
+import { Mongo } from 'meteor/mongo';
 import tls from 'tls';
 // FIX For TLS error see more here https://github.com/RocketChat/Rocket.Chat/issues/9316
 // TODO: Remove after NodeJS fix it, more information https://github.com/nodejs/node/issues/16196 https://github.com/nodejs/node/pull/16853
 tls.DEFAULT_ECDH_CURVE = 'auto';
 
+// Revert change from Meteor 1.6.1 who set ignoreUndefined: true
+// more informatin https://github.com/meteor/meteor/pull/9444
+Mongo.setConnectionOptions({
```

Comment:
May I ask why this is being done in the CORS package? That's my only conundrum with this..

Reply:
My only guess is because it is the first one to be loaded

Reply:
https://github.com/meteor/meteor/issues/7455